### PR TITLE
Fix syntax in query examples

### PR DESCRIPTION
**Pull request by manototh**

### CODE REVIEW

File: restapi/query.mdx [hunk 1]
```diff
@@ -303,7 +303,7 @@ curl -X 'POST' 'https://api.axiom.co/v1/datasets/_apl?format=tabular' \
 -H 'Accept-Encoding: gzip' \
 -H 'Content-Type: application/json' \
 -d '{
-      "apl": "github-issue-comment-event | where repo startswith \"kubernetes/\" | where actor !endswith \"[bot]\" | summarize dcount(actor) by bin_auto(_time)",
+      "apl": "\"github-issue-comment-event\" | where repo startswith \"kubernetes/\" | where actor !endswith \"[bot]\" | summarize dcount(actor) by bin_auto(_time)",
```
Comment (mhr3) [inline restapi/query.mdx]: missing brackets

File: restapi/query.mdx [hunk 2]
```diff
@@ -318,7 +318,7 @@ curl -X 'POST' 'https://api.axiom.co/v1/datasets/_apl?format=tabular' \
 -H 'Accept-Encoding: gzip' \
 -H 'Content-Type: application/json' \
 -d '{
-      "apl": "github-issues-event | where (actor !endswith \"[bot]\" and repo startswith \"cockroachdb/\" and actor !~ \"cockroach-teamcity\") | summarize topk(actor, 5) by bin_auto(_time), action",
+      "apl": "\"github-issues-event\" | where (actor !endswith \"[bot]\" and repo startswith \"cockroachdb/\" and actor !~ \"cockroach-teamcity\") | summarize topk(actor, 5) by bin_auto(_time), action",
```
Comment (mhr3) [inline restapi/query.mdx]: ditto

**Review by mhr3 — COMMENTED** [review]
Also noticed the Authorization header is using `-H 'Authorization: Bearer $API_TOKEN'` which is wrong, if the token was stored in an env, the use of `'` would prevent expanding it and a literal `$API_TOKEN` would be sent as a token.

File: restapi/query.mdx (hunk 2 above)
Comment (ted-gould) [reply restapi/query.mdx]: The parens in the where aren't really needed.

File: restapi/query.mdx (hunk 1 above)
Comment (manototh) [inline restapi/query.mdx]:
```suggestion
      "apl": "[\"github-issue-comment-event\"] | where repo startswith \"kubernetes/\" | where actor !endswith \"[bot]\" | summarize dcount(actor) by bin_auto(_time)",
```

File: restapi/query.mdx (hunk 2 above)
Comment (manototh) [inline restapi/query.mdx]:
```suggestion
      "apl": "[\"github-issues-event\"] | where actor !endswith \"[bot]\" and repo startswith \"cockroachdb/\" and actor !~ \"cockroach-teamcity\" | summarize topk(actor, 5) by bin_auto(_time), action",
```

**Comment by manototh** [comment]
@mhr3 @ted-gould Thanks a lot both! I think this should be fine now. Could you please take another look?

**Comment by manototh** [comment]
> Also noticed the Authorization header is using `-H 'Authorization: Bearer $API_TOKEN'` which is wrong, if the token was stored in an env, the use of `'` would prevent expanding it and a literal `$API_TOKEN` would be sent as a token.

Well spotted. Corrected. I would like to move towards using `API_TOKEN` as placeholder everywhere in the docs. This is in line with the Google Developer style guide and it leaves the issue of using env vars to the user.

**Review by mhr3 — APPROVED** [review]

**Pull request merged**